test/data/users.js

```diff
@@ -40,7 +37,10 @@ export const extendedUsers = dataStore({
       : (inPast ? fakePastDate([lastCreatedAt]) : new Date().toISOString()),
     updatedAt: null,
     deletedAt,
-    preferences,
+    preferences: {
+      site: preferences?.site ?? {},
+      projects: preferences?.projects ?? {}
+    }
```

Comment:
I've set it up this way so that it's easy for future component tests that only need to set a single site preference or a single project preference. For example, instead of:

```js
testData.extendedUsers.createPast(1, { site: { projectSortMode: 'latest' }, projects: {} });
```

We can omit the empty `projects` object:

```js
testData.extendedUsers.createNew({ site: { projectSortMode: 'latest' } });
```